packages/amplify-frontend-ios/native-bindings-codegen/templates/preamble.jst

```diff
@@ -22,5 +22,5 @@ const path = require('path');
 const { pathManager } = require('@aws-amplify/amplify-cli-core');
 
 const BINARY_PATH = 'resources/amplify-xcode';
-const PACKAGE_NAME = 'amplify-frontend-ios';
+const PACKAGE_NAME = '@aws-amplify/amplify-frontend-ios';
```

Comment:
why this change ? was it missed in the @ aws-amplify refactor?

Reply:
No. it was hand-edited here https://github.com/aws-amplify/amplify-cli/blob/6c43a022754a196f549724336405e44bb2606c7a/packages/amplify-frontend-ios/lib/amplify-xcode.js#L25 when we added prefix.
But we didn't realize this code is generated, so this change fixes the generator.